README.md

```diff
@@ -99,31 +99,6 @@ Deploy the stack
 cdk deploy
 ```
 
```

Comment:
Removing it? What is the reason?

Reply:
Sorry this was a mistake - I went ahead and pushed the updated changes to the README